### DISCUSSION TITLE
sqlite question

### DISCUSSION THREAD

**Discussion by nanr** [E3SM-Project/zstash#324] [Questions]
### Question criteria

- [X] I added a descriptive title here.
- [X] I searched the [zstash GitHub Discussions](https://github.com/E3SM-Project/zstash/discussions) to find a similar question and didn't find it.
- [X] I searched the [zstash documentation](https://e3sm-project.github.io/zstash).

### What is the deadline?

Urgent.   

### Describe your question

I am having trouble updating a zstash archive that seems to be corrupted.  

### What machine were you running on?

perlmutter

### Environment

I'm not sure what zstash version I'm using.  Here is the environment I'm loading.

/global/common/software/e3sm/anaconda_envs/load_latest_e3sm_unified_pm-cpu.sh

### Minimal Complete Verifiable Example (MCVE)

```python
(e3sm_unified_1.9.1_login) nanr@perlmutter:login23:/global/cfs/cdirs/mp9/archive/E3SMv2-SMBB/v2.LR.SSP370-smbb_0111/zstash> sqlite3 index.db
SQLite version 3.44.0 2023-11-01 11:23:50
Enter ".help" for usage hints.
**sqlite> delete from tars where name="000041.tar";
Parse error: no such column: 000041.tar
  delete from tars where name="000041.tar";
                error here ---^**
sqlite> delete from files where tar="000041.tar";
Parse error: no such column: 000041.tar
  delete from files where tar="000041.tar";
                error here ---^
```

**Comment by nanr** [comment]
Fix was easy.

sqlite3 index.db
delete from tars where name='000041.tar';
delete from files where tar='000041.tar';
.exit

**Reply by forsyth2** [reply]
I'm glad the fix was easy. Please let us know if you run into further issues.

Sorry for not responding sooner; I was out last week.
